modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/RelationshipObjectFieldBusinessType.java

```diff
@@ -123,7 +123,13 @@ public Object getValue(ObjectField objectField, Map<String, Object> values)
 				ObjectRelationshipConstants.TYPE_ONE_TO_MANY) ||
 			values.containsKey(objectField.getName())) {
 
-			return values.get(objectField.getName());
+			Object value = values.get(objectField.getName());
+
+			if (Validator.isNull(value)) {
+				return 0;
```

Comment:
We need a 0 in order for the service to override/empty the current relatioship value. This value is returned to DefaultObjectEntryManagerImpl.java and if it is null nothing is done:

https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java#L1026

```
			if ((value == null) && !objectField.isRequired()) {
				continue;
			}
```
We need to set it empty.

Reply:
> We need a 0 in order for the service to override/empty the current relatioship value

@crodriguezy Can you show me where this logic is? Thanks!

Reply:
@crodriguezy `values.get(objectField.getName())` will return `null` if the user does not send any value for this field, so I believe that this behavior is correct, we should not change the value in this case.
We should only change it if the value is different from `null` (an empty value, for example) .

Reply:
Hey @carolmariaabb , it's null what's coming from the above call in the method:

Object value = ObjectEntryValuesUtil.getValue(
				_objectDefinitionLocalService, _objectEntryLocalService,
				objectField, _objectFieldBusinessTypeRegistry, userId,
				objectEntry.getProperties());
And according to my tests sending a 0 would make no difference for a valid case (when the user do not inform it through the REST api). 
You can contact me via slack and we could discuss what would be a better approach to handle this case.
Thank you!

Reply:
@crodriguezy imagine the use case: The user adds a new entry (via Rest API) with a valid `ID` for the relationship field. After that, the user tries to update (via Rest API) a different field in this entry and he doesn't send any value for the relationship field. We shouldn't override its value in this case, as it's the expected behavior for all the other fields.

Reply:
I see. We need to set a different value when we identify this case then (0 or empty) in previous layers/calls. Where do you suggest we make the change?

Reply:
@crodriguezy you can try to change the `onChange` logic [in this file](https://github.com/liferay/liferay-portal/blob/045dfcd7cf0f5630008e30b11e4694570adea96a/modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ObjectRelationship/ObjectRelationship.tsx#L200).. You can send 0 or empy instead of null..

Reply:
Thanks @carolmariaabb . Please see the changes, where I removed last commit and modified the onChange method instead. I tested the REST API and it's not changing the values if I don't send any for the relationships.

Reply:
Hi @crodriguezy, could you change [here](https://github.com/liferay-objects/liferay-portal/blob/880e684d446147e803b0434ad632a69a9938ac11/modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ObjectRelationship/ObjectRelationship.tsx#L302-L307) to?

Reply:
@crodriguezy Also, wouldn't it be better to send an empty string instead of zero? Considering the case where the field is required, I was wondering how it will behave with zero.

Reply:
@carolmariaabb please see the force-pushed changes in last commit